example.py

```diff
@@ -135,12 +135,41 @@ def world_command_examples():
 
 def editor_example():
     """This editor example shows how to interact with holodeck worlds while they are being built
-    in the Unreal Engine. Most people that use holodeck will not need this.
+    in the Unreal Engine Editor. Most people that use holodeck will not need this.
+
+    This example uses a custom scenario, see 
+    https://holodeck.readthedocs.io/en/latest/usage/examples/custom-scenarios.html
+
+    Note: When launching Holodeck from the editor, press the down arrow next to "Play" and select
+    "Standalone Game", otherwise the editor will lock up when the client stops ticking it.
     """
-    agent_sensors = [sensors.RGBCamera, sensors.LocationSensor, sensors.VelocitySensor]
-    agent = AgentDefinition("uav0", agents.UavAgent, agent_sensors)
-    env = HolodeckEnvironment([agent], start_world=False)
-    env.agents["uav0"].set_control_scheme(1)
+
+    config = {
+        "name": "test",
+        "world": "TestWorld",
+        "main_agent": "uav0",
```

Comment:
We updated the config files to require package_name. Has that change been merged yet and should be reflected here? Just checking.

Reply:
It does not need to be here. The package name is only used when loading a binary, which requires the scenario to know which package we are loading up. Here there is no binary, only the unreal editor.

Reply:
Basically, that is only needed when specifying a configuration that isn't a JSON file, correct @daniekpo ?